### ISSUE TITLE
[gatsby-source-wordpress] Infinity download of media files

### ISSUE THREAD

**Issue by SilencerWeb** [gatsbyjs/gatsby#16213]
## Description
Right now I have 45 media files (4 PDFs and 41 images) on WordPress server. When I'm starting either `npm run develop` or `npm run build` and media files are downloading, they never reach the end. It just stops on either 41/45 or 42/45 or 44/45 and that's all, it can be like at for few hours and nothing will happen.

Is there anything you can do about it? Maybe throw an error after a certain delay? Download media files one after each other?

Also, another thing you can do is cache file right after downloading it. Today I tried to download media files a lot of times and I noticed that they are getting cached only after successful downloading of all media files.

### Steps to reproduce
1. Set up a WordPress server with a lot of media files.
2. Find slow and not stable internet.
3. Run either `npm run develop` or `npm run build`.
I understand that it's too hard to do so I'm willing to help. Unfortunately, can't give access to the database :(

### Expected result
Successful download of all media files.

### Actual result
Infinity downloading of media files.

### Environment
```  
  System:
    OS: macOS 10.14.5
    CPU: (4) x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 12.6.0 - /var/folders/n0/5tbk0_4x6j9c2v77dzwxq2mr0000gn/T/yarn--1564488855130-0.6273237784710566/node
    Yarn: 1.17.3 - /var/folders/n0/5tbk0_4x6j9c2v77dzwxq2mr0000gn/T/yarn--1564488855130-0.6273237784710566/yarn
    npm: 6.10.2 - /usr/local/bin/npm
  Languages:
    Python: 2.7.10 - /usr/bin/python
  Browsers:
    Chrome: 75.0.3770.142
    Safari: 12.1.1
  npmPackages:
    gatsby: ^2.11.7 => 2.13.43
    gatsby-background-image: ^0.7.6 => 0.7.6
    gatsby-image: ^2.2.3 => 2.2.7
    gatsby-plugin-mailchimp: ^5.1.0 => 5.1.2
    gatsby-plugin-manifest: ^2.2.0 => 2.2.4
    gatsby-plugin-offline: ^2.2.0 => 2.2.4
    gatsby-plugin-react-helmet: ^3.1.0 => 3.1.2
    gatsby-plugin-react-svg: ^2.1.1 => 2.1.1
    gatsby-plugin-remove-trailing-slashes: ^2.1.0 => 2.1.2
    gatsby-plugin-sass: ^2.1.0 => 2.1.3
    gatsby-plugin-sharp: ^2.2.1 => 2.2.9
    gatsby-plugin-sitemap: ^2.2.3 => 2.2.4
    gatsby-source-filesystem: ^2.1.1 => 2.1.6
    gatsby-source-wordpress: ^3.1.2 => 3.1.11
    gatsby-transformer-sharp: ^2.2.0 => 2.2.5
```
Tried to upgrade dependncies, didn't help. Actually, everything got even worse.

**Comment by Polyhistor** [comment]
Does it get stuck on Gatsby transform nodes?

**Comment by SilencerWeb** [comment]
@Polyhistor, yes, like this:

![image](https://user-images.githubusercontent.com/20713191/62181832-08632300-b36e-11e9-9ff1-27271b23ff9b.png)

**Comment by SilencerWeb** [comment]
Closing as duplicate of #6654.

For those who ran into this issue, try to set `GATSBY_CONCURRENT_DOWNLOAD` to any number you want, default is 200.

Examples:
**Linux/mac:**
`GATSBY_CONCURRENT_DOWNLOAD=5 gatsby build`
**Windows:**
`setx GATSBY_CONCURRENT_DOWNLOAD 5; gatsby build`

**Comment by hannah4669** [comment]
> Closing as duplicate of #6654.
> 
> For those who ran into this issue, try to set `GATSBY_CONCURRENT_DOWNLOAD` to any number you want, default is 200.
> 
> Examples:
> **Linux/mac:**
> `GATSBY_CONCURRENT_DOWNLOAD=5 gatsby build`
> **Windows:**
> `setx GATSBY_CONCURRENT_DOWNLOAD 5; gatsby build`

um where do you set it? I'm new to gatsby and I have no idea where
